worker/src/main/java/gov/cms/ab2d/worker/adapter/bluebutton/BeneficiaryAdapterStub.java

```diff
@@ -33,6 +32,8 @@ public GetPatientsByContractResponse getPatientsByContract(String contractNumber
         final int contractSno = extractContractSno(contractNumber);
         final int startOffset = (contractSno - 1) * 100;
 
+        log.info("Getting patients at offset {}", startOffset);
```

Comment:
This stub class is temporary. No need to log anything; you can revert changes to this class.